source/change-streams/change-streams.rst

```diff
@@ -260,16 +260,15 @@ Driver API
 
   class ChangeStreamOptions {
     /**
-     * Allowed values: ‘default’, ‘updateLookup’.  Defaults to ‘default’.  When set to
-     * ‘updateLookup’, the change notification for partial updates will include both
-     * a delta describing the changes to the document, as well as a copy of the entire
-     * document that was changed from some time after the change occurred.  For forward
-     * compatibility, a driver MUST NOT raise an error when a user provides an unknown
-     * value. The driver relies on the server to validate this option.
-     *
+     * Allowed values: ‘updateLookup’.  When set to ‘updateLookup’, the change notification
+     * for partial updates will include both a delta describing the changes to the document,
+     * as well as a copy of the entire document that was changed from some time after the 
+     * change occurred. The default is to not send a value. 
+     * For forward compatibility, a driver MUST NOT raise an error when a user provides an
+     * unknown value. The driver relies on the server to validate this option.
      * @note this is an option of the `$changeStream` pipeline stage.
      */
-    fullDocument: string = ‘default’;
```

Comment:
I left a comment in [SPEC-1350](https://jira.mongodb.org/browse/SPEC-1350) to ask if _all_ server versions since 3.6.0 allow this.

Beyond that, I think this warrants a bump to spec document's minor version (1.6.1 -> 1.7.0) in the document header, as well as a change log entry near the end of the document.

Also, this will warrant removing the `fullDocument` field (within `$changeStream`) from all APM assertions within the spec tests. AFAIK, that should instruct test runners to omit any check for the field (while still allowing its presence if drivers always send "default") per the [Spec Test Match Function](https://github.com/mongodb/specifications/tree/master/source/change-streams/tests#spec-test-match-function) rules.